develop/devguide/Connector/ConnectorHelp/Connector_help_pages.md

```diff
@@ -0,0 +1,144 @@
+---
+uid: Connector_help_pages
+---
+
+# Connector help pages
+
+For every connector that is developed for DataMiner, a corresponding help page should be available. This help page details among others how the connections should be set up when you create an element with the connector, how the element should be used, etc. If a connector exports child connectors, a help page should also be created for each of these child connectors.
+
+When you develop a connector using DIS, you can generate the connector help with the *Plugins* > *Generate driver help* option in the [DIS menu](xref:DIS_menu). You can also create the connector help directly in Markdown, starting from our [template](xref:Connector_help_template). When you have created the help page, add it in the [dataminer-docs-connectors](https://github.com/SkylineCommunications/dataminer-docs-connectors) repository.
+
+## Writing connector help
+
+When you write connector help, keep the instructions below in mind.
+
+### Markdown syntax
+
+Use DocFX Flavored Markdown (DFM). See [Markdown syntax](xref:contributing#markdown-syntax).
+
+### Title
+
+The **title** of the help page must be the **exact name of the connector**.
+
+It is important that this is the same name as in the [DataMiner Catalog](https://catalog.dataminer.services/), so that the help page can automatically be displayed there.
```

Comment:
Having the title being exactly the same is not needed anymore to embed it into catalog. 
Instead, the following is mandatory regarding the .md file: 
1. it should be under /dataminer-docs-connectors/connector_help/connector-help
2. the file name should be the name of the connector where the spaces are replaced with underscore
The above two points probably don't fit under the Title section, but I do think we need to mention this somewhere on this page.

Reply:
I've changed this so the points you mentioned are now included.